lib/liquidTestRunner.js

```diff
@@ -409,9 +413,12 @@ async function runTestsWithOutput(
 // RETURN (AND LOG) ONLY PASSED OR FAILED
 // CAN BE USED BY GITHUB ACTIONS
 async function runTestsStatusOnly(firmId, handle, testName = "") {
-  const testRun = await runTests(firmId, handle, testName, false, "none");
-  if (testRun && testRun.status === "completed") {
-    const errorsPresent = checkAllTestsErrorsPresent(testRun.tests);
+  const testsRun = await runTests(firmId, handle, testName, false, "none", true);
```

Comment:
@Woetfin why the extra `true`?
`runTests` has 5 parameters but here we are passing 6 arguments

Reply:
was for the skipSpinner